tshirts.cpp

```diff
@@ -17,6 +17,9 @@ int main() {
     assert(size(37) == 'S');
     assert(size(40) == 'M');
     assert(size(43) == 'L');
+
+    assert(size(38) == 'M');

```

Comment:
What's the reason to choose `M` for 38, instead of `S`?
How would this relate to a customer experience?